backend/src/org/commcare/session/CommCareSession.java

```diff
@@ -773,6 +843,11 @@ public boolean isViewCommand(String command) {
         return entries.size() == 1 && entries.elementAt(0).isView();
     }
 
+    public boolean isSyncCommand(String command) {
+        Vector<Entry> entries = this.getEntriesForCommand(command);
+        return entries.size() == 1 && entries.elementAt(0) instanceof SyncEntry;
```

Comment:
Is there a reason we're doing this with a type check rather than a polymorphic check(IE: "entries.elementAt(0).isSync() or (0).getCommandType().equals(TYPE_SYNC))? The latter would be more "Best practice"y.